ai/commands.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import subprocess
+import sys
+
+from .network import Network
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+formatter = logging.Formatter(
+    "%(asctime)s - %(name)-11s - %(levelname)-8s - %(message)s",
+    datefmt="%H:%M:%S",
+)
+
+stream_handler = logging.StreamHandler()
+stream_handler.setFormatter(formatter)
+
+logger.addHandler(stream_handler)
+
+
+class Commands:
+    def __init__(self, network: Network, team_name: str):
+        self.network = network
+        self.team_name = team_name
+
+    def __send_command(self, command: str) -> str:
+        logger.debug(f"Sending: {command}")
```

Comment:
Well, this is a bit of an obscure fact, but `logging` it meant to use `%` formatters

> https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles